diff --git a/node_modules/react-native-audio-analyzer/react-native-audio-analyzer.podspec b/node_modules/react-native-audio-analyzer/react-native-audio-analyzer.podspec
index 044fd99..3876368 100644
--- a/node_modules/react-native-audio-analyzer/react-native-audio-analyzer.podspec
+++ b/node_modules/react-native-audio-analyzer/react-native-audio-analyzer.podspec
@@ -15,6 +15,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/exzos28/react-native-audio-analyzer.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm}", "cpp/**/*.{hpp,cpp,c,h}"
+  s.public_header_files = "cpp/AudioAnalyzer.h"  
 
   s.dependency "ffmpeg-kit-ios-audio", "5.1"
 
