diff --git a/node_modules/react-native-modal/dist/utils.js b/node_modules/react-native-modal/dist/utils.js
index 2868f79..b0b07c1 100644
--- a/node_modules/react-native-modal/dist/utils.js
+++ b/node_modules/react-native-modal/dist/utils.js
@@ -17,12 +17,17 @@ export const initializeAnimations = () => {
     animatable.initializeRegistryWithDefinitions(animationDefinitions);
 };
 export const makeSlideTranslation = (translationType, fromValue, toValue) => ({
-    from: {
+    0:{
         [translationType]: fromValue,
+        ['scaleX']:0
     },
-    to: {
-        [translationType]: toValue,
+    0.5:{
+        ['scaleX']:0.5
     },
+    1:{
+        [translationType]: toValue,
+        ['scaleX']:1
+    }
 });
 // User can define custom react-native-animatable animations, see PR #72
 // Utility for creating our own custom react-native-animatable animations
